src/v2/Apps/Consign/Routes/SubmissionFlow/Mutations/removeAssetFromConsignmentSubmission.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  removeAssetFromConsignmentSubmissionMutation,
+  removeAssetFromConsignmentSubmissionMutationResponse,
+  RemoveAssetFromConsignmentSubmissionInput,
+} from "v2/__generated__/removeAssetFromConsignmentSubmissionMutation.graphql"
+import { commitMutation, graphql } from "relay-runtime"
+
+export const removeAssetFromConsignmentSubmission = (
+  relayEnvironment,
+  input: RemoveAssetFromConsignmentSubmissionInput
+) => {
+  return new Promise<removeAssetFromConsignmentSubmissionMutationResponse>(
+    (resolve, reject) => {
+      commitMutation<removeAssetFromConsignmentSubmissionMutation>(
+        relayEnvironment,
+        {
+          mutation: graphql`
+            mutation removeAssetFromConsignmentSubmissionMutation(
+              $input: RemoveAssetFromConsignmentSubmissionInput!
+            ) {
+              removeAssetFromConsignmentSubmission(input: $input) {
+                asset {
+                  submissionID
+                }
+              }
+            }
+          `,
+          variables: {
+            input: {
+              ...input,
+              clientMutationId: Math.random().toString(8),
+            },
+          },
+          onError: reject,
+          onCompleted: (response, errors) => {
+            if (errors && errors.length > 0) {
+              reject(new Error(JSON.stringify(errors)))
+            } else {
+              resolve(response)
+            }
+          },
+        }
+      )
+    }
+  )
+}
```

Comment:
You may want to look at https://github.com/artsy/force/blob/main/src/v2/Utils/Hooks/useMutation.ts

An example: https://github.com/artsy/force/blob/5b6a9d3b30e5c8039aa2388b80b4cafc7ee63d7d/src/v2/Apps/Settings/Routes/EditSettings/Components/useUnlinkSettingsLinkedAccount.ts

Reply:
👀  Looks interesting!